code/index.html

```diff
@@ -2,17 +2,131 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project Name</title>
+    <title>The Cat Working Space</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    
   </head>
   <body>
+
     <header>
-      <!-- If you want a header with a title, logo and a navbar, here is where you should add that. -->
+
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+      <img src="/assets/logo.png" alt= "Katze">
+
+      <nav>
+        <a href="#"> Reservation</a>
+        <a href="#"> Über</a>
+        <a href="#"> Kontakt</a>
+      </nav>
+    
     </header>
 
+    <section class="first-container">
```

Comment:
> Also want to know, if there is a better solution than to call the .first-container .container like this. I adapted the style from the code-along-video. But it confused me. How can i rename it to understand it better - me and others?

When programming, naming things in a good way is one of the hardest things to achieve! I think it's great that you are reflecting on this. We have a section about it in our [code guidelines ](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) that you can check out. Maybe it could be called `title-container` or `heading-container` since it's containing your `h1`?